core/src/main/java/org/apache/carbondata/core/locks/HdfsFileLock.java

```diff
@@ -47,26 +44,12 @@
 
   private static String tmpPath;
 
-  static {
```

Comment:
Use the mdtfile path to use systemlocking

Reply:
fixed